core/domain/training_job_result_domain_test.py

```diff
@@ -37,9 +37,9 @@ def test_validate_job_data_with_valid_model_does_not_raise_exception(self): # py
             job_id, algorithm_id, classifier_data)
         job_result.validate()
 
-    def test_validate_job_data_with_invalid_model_raises_exception(self):
-        """Ensure that validation checks raise exception when
-        an invalid classifier model is supplied.
+    def test_job_data_with_invalid_job_id_raises_exception_on_validtion(self):
+        """Ensure that validation checks raise exception when an invalid
```

Comment:
Here & below: probably don't need the doc comment anymore since the test name is pretty straightforward.

Reply:
It becomes necessary because of the pylint. The doc comment option hasn't been disabled globally.

Reply:
Ah, thanks for clarifying.